README.md

```diff
@@ -5,177 +5,144 @@
 
 Template github repo used for new Typescript based projects.
 
-# Instructions
+## Get started
 
-If this is a HMPPS project then the project will be created as part of bootstrapping -
-see https://github.com/ministryofjustice/hmpps-project-bootstrap. You are able to specify a template application using
-the `github_template_repo` attribute to clone without the need to manually do this yourself within GitHub.
+Dev: https://github.com/ministryofjustice/hmpps-suicide-risk-form-ui
 
-This project is community managed by the mojdt `#typescript` slack channel.
-Please raise any questions or queries there. Contributions welcome!
+### Pre-requisites
 
-Our security policy is located [here](https://github.com/ministryofjustice/hmpps-suicide-risk-form-ui/security/policy).
+You'll need to install:
 
-More information about the template project including features can be
-found [here](https://dsdmoj.atlassian.net/wiki/spaces/NDSS/pages/3488677932/Typescript+template+project).
+* [Node 22.x](https://nodejs.org/download/release/latest-v22.x)*
+* [Docker](https://www.docker.com/)
+* [Java 21](https://www.oracle.com/uk/java/technologies/downloads/)**
```

Comment:
Technically the Java version doesn't matter any more, as you're using WireMock in Docker instead of npm.